src/Avalonia.FuncUI/DSL/Buttons/ToggleButton.fs

```diff
@@ -27,6 +27,9 @@ module ToggleButton =
         [<Obsolete "use 'onChecked' or 'onUnchecked' instead">]
         static member onIsCheckedChanged<'t when 't :> ToggleButton>(func: Nullable<bool> -> unit, ?subPatchOptions) : IAttr<'t> =
```

Comment:
I think we should get rid of this method avoid confusion.

Reply:
done